doc/configuration.rst

```diff
@@ -1573,6 +1573,25 @@ Arguments:
     same as the fastboot default, which is computed after querying the target's
     ``max-download-size`` variable.
 
+DFUDriver
+~~~~~~~~~
+A DFUDriver allows the upload of images to a device in DFU (Device Firmware
```

Comment:
s/download/upload/

Reply:
Right, I got confused by the upload/download terms in dfu-util. I'll fix it.

Reply:
Fixed.